internal/tsetse/plugin.ts

```diff
@@ -0,0 +1,41 @@
+import * as tsForTypes from 'typescript';
+import * as tsServerLibForTypes from 'typescript/lib/tsserverlibrary';
+
+import * as perfTrace from '../tsc_wrapped/perf_trace';
+import * as pluginApi from '../tsc_wrapped/plugin_api';
+
+import {Checker} from './checker';
+import {registerRules} from './runner';
+
+// Installs the Tsetse language server plugin, which checks Tsetse rules in your
+// editor and shows issues as semantic errors (red squiggly underline).
+
+function init() {
+  return {
+    create(info: ts.server.PluginCreateInfo) {
+      const oldService = info.languageService;
+      const checker = new Checker(oldService.getProgram());
+      // Add disabledRules to tsconfig to disable specific rules
+      // "plugins": [
+      //   {"name": "...", "disabledRules": ["equals-nan"]}
```

Comment:
this is troubling, how would I keep that list in sync with the disabled state of the rules under `tsc`?
Seem like we should read from `bazelOpts.disabledTsetseRules` but TS doesn't let plugins see the whole tsconfig? Should we move the other config under the "plugins" section?

Reply:
TS doesn't give us direct access to the tsconfig, but of course we can just parse it ourselves -- added code to do that, wdyt? (It only runs once on startup of vscode)

(I wasn't aware of the bazelOpts thing)